docs/_interface/metrics.md

```diff
@@ -0,0 +1,102 @@
+---
+title: Metrics
+sort_key: C
+---
+
+## Meters and Timers
+
+Note, metrics collection is disabled by default in order not to incur reporting overhead for the average user. To enable it please specify the `--metrics` flag to geth.
+
+Geth has quite a nice logging system, capable of creating leveled log entries tagged with various parts of the system. This helps enormously during debugging to see exactly what the system is doing, what branches it's taking, etc. However, logs are not particularly useful when the system does work correctly, just not very optimally: one - or even a  handful - of logged events is not really statistically relevant, and tracing more in log files can quickly become unwieldy.
```

Comment:
I dunno, maybe we can skip this? It's a page about metrics, I don't think we need to justify why metrics are different/better than logs.